src/NuGet.Core/NuGet.PackageManagement/Utility/LogUtility.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System.Collections.Generic;
+using NuGet.Commands;
 using NuGet.Common;
 using NuGet.ProjectManagement;
 using NuGet.ProjectModel;
```

Comment:
Should these be in alphabetical order?

Reply:
These are added "automatically" and looking at other files there is no order enforced. I'll reorganize them but maybe we should consider having a team agreement about this.

Reply:
If you use Visual Studio's "sort and remove unused usings", it should put the `System` namespaces first, but otherwise sort them alphabetically

https://github.com/NuGet/NuGet.Client/blob/3b6ddc5b9f878f7408695f16c67198fa4e747b71/.editorconfig#L41